client/api/service.py

```diff
@@ -0,0 +1,41 @@
+from FileReader.DataInstance import DataInstance
+# from input.Model import Model
+
+from ipaddress import ip_address
+from urllib.parse import urlparse
+
+from client.api.request import Request
+from client.api.connection_handler import ConnectionHandler
+
+
+class Service:
+
+    def __init__(self, address):
+        ip, port = self.parse_ip_port(address)
+        self.handler = ConnectionHandler(ip, port)
+
+    def predict(self, model: Model, data_instance: DataInstance):
+        request = Request(model, data_instance)
+        result = self.handler.send(request, "/predict")
+        print(result)
```

Comment:
Też raczej zostanie zmienione w przyszłości. Na pewno chcielibyśmy gdzieś to zapisywać. Możliwe, że zrobimy dodatkowy plik `main.py` czy coś w tym stylu z `ArgumentParser`, żeby dało się z terminalu to wywołać. A może po prostu tu dodamy jakąś opcję output'u